python/sglang/srt/layers/attention/triton_ops/decode_attention.py

```diff
@@ -402,7 +402,7 @@ def _decode_grouped_att_m_fwd(
     sm_scale,
     logit_cap,
 ):
-    BLOCK = 32
+    BLOCK = 16 if is_hip() else 32
```

Comment:
we should not cut by half for HIP globally here.

Reply:
it doesn't work well in latest vllm with BLOCK 32